tests/test_local.py

```diff
@@ -14,23 +14,34 @@ async def test_list(self) -> None:
         adapter = LocalAdapter()
         items = await adapter.list(path.join(path.dirname(__file__), "resources"))
 
-        assert len(items) == 2
+        assert len(items) == 3
 
-        file_found = False
+        file_found_1mb = False
+        file_found_10mb = False
         directory_found = False
         for item in items:
             if isinstance(item, LocalFile):
-                assert item.path == path.join(
+                if item.path == path.join(
                     path.abspath(path.dirname(__file__)), "resources", "1mb.bin"
-                )
-                assert await item.size == 1048576
-                file_found = True
+                ):
+                    assert await item.size == 1048576
+                    file_found_1mb = True
+                elif item.path == path.join(
+                    path.abspath(path.dirname(__file__)), "resources", "10mb.bin"
```

Comment:
Niet breaking: de enige optimalisatie die ik zo kan vinden is dat 'resources' wellicht een constante kan zijn